eras/shelley/impl/src/Cardano/Ledger/Shelley/HardForks.hs

```diff
@@ -65,3 +66,8 @@ translateUpperBoundForPlutusScripts ::
   ProtVer ->
   Bool
 translateUpperBoundForPlutusScripts pv = pvMajor pv > natVersion @8
+
+forgoPointerAddressResolution ::
+  ProtVer ->
+  Bool
+forgoPointerAddressResolution pv = pvMajor pv > natVersion @9
```

Comment:
```suggestion
forgoPointerAddressResolution pv = pvMajor pv > natVersion @8
```

Reply:
We can stop counting pointer stake starting with the Conway era, ie version 9

Reply:
Oops :facepalm:   Thanks!

Reply:
IMHO using `>=` would make this a bit more obvious. Not suggesting changing it though, unless we change it in the whole module, cause otherwise  inconsistency would make it even more confusing.

Reply:
I agree, the number (8 or 9) is a stronger signal that stays in your mind than the operator is, so you associate it with the number . Happy to change it in another commit